boards/arm/stm32/stm32ldiscovery/configs/chrono/defconfig

```diff
@@ -20,7 +20,7 @@ CONFIG_ARCH_CHIP_STM32=y
 CONFIG_ARCH_CHIP_STM32L152RB=y
 CONFIG_ARCH_IRQBUTTONS=y
 CONFIG_ARCH_STACKDUMP=y
-CONFIG_ARM_LWL_CONSOLE=y
+CONFIG_LWL_CONSOLE=y
```

Comment:
need run tools/refresh.sh --silent

Reply:
Thank you @xiaoxiang781216 I just did it!